.github/workflows/cli-tests.yml

```diff
@@ -35,9 +35,15 @@ jobs:
       ## Install pip
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.7.x'
+          python-version: '3.9.x'
```

Comment:
dbt 1.6 dropped support for python 3.7

Reply:
do you know what we're running in the container? It looks like we're not pinning just taking the bundled debian version

Reply:
Python 3.9.2